.flowconfig

```diff
@@ -19,6 +19,9 @@
 ; Ignore metro
 .*/node_modules/metro/.*
 
+; Web + RN TODO temp fix. Duplicated providers.
```

Comment:
Could you please create an issue for it and then merge that PR?

Reply:
sure

Reply:
https://github.com/kiwicom/margarita/issues/77 done